File: question5.py
from datetime import datetime
# dogum tarihi girilen kisinin yasini hesaplayan kod

dYil=int(input("Dogum yilinizi giriniz:"))

mTarih=datetime.now()
mYil=mTarih.year

yas = mYil-dYil

print("Yasiniz:" + str(yas))

File: question5.java
import java.util.Scanner;
import java.util.Calendar;
public class question5 {
    public static void main(String[] args) {
        // dogum tarihi girilen kisinin yasini hesaplayan kod

        Scanner scanner = new Scanner(System.in);

        System.out.println("Dogum tarihini giriniz:");

        int dYil=scanner.nextInt();

        Calendar calendar=Calendar.getInstance();

        //mevcut yili alalim

        int mYil=calendar.get(Calendar.YEAR);

        //yas hesabi

        int yas = mYil-dYil;

        System.out.println("Yasiniz:" + yas);

        scanner.close();
        
    }
}
